### ISSUE TITLE
Code Table Request - REMOVE ectoparasite examination

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#6220]
## Initial Request

_**Goal**: Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

[ectoparasite examination](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#ectoparasite_examination)  should be merged into some combination of

* https://arctos.database.museum/info/ctDocumentation.cfm?table=CTATTRIBUTE_TYPE&field=detected#examined_for
* https://arctos.database.museum/info/ctDocumentation.cfm?table=CTATTRIBUTE_TYPE&field=detected#detected
 * https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#not_detected

Migration path and value needs determined; I don't understand these data. 

This will likely involve another code table request for a new value in https://arctos.database.museum/info/ctDocumentation.cfm?table=ctexamined_detected

Data:
[temp_epa.csv.zip](https://github.com/ArctosDB/arctos/files/11383592/temp_epa.csv.zip)

Summary:

```

 guid_prefix | count 
-------------+-------
 UNR:Mamm    |     3
 ASNHC:Mamm  |     4
 DMNS:Bird   |  6088
 UTEP:Mamm   |     1
 NMU:Mamm    |  1625
 DMNS:Herp   |     2
 DMNS:Mamm   |  3606
 MLZ:Mamm    |    35
 MSB:Mamm    | 31880
 CHAS:Mamm   |     1
 MSB:Herp    |    11
 CHAS:Teach  |     2
 UNCG:Mamm   |   694
 MSB:Host    |    71
 UMZM:Mamm   |    22
 UMNH:Mamm   |   832
 MSB:Bird    |     6
 CHAS:Bird   |     3
 UWYMV:Bird  |     5
 UMNH:Herp   |     4
 UMZM:Bird   |    90
 UMNH:Bird   |     2
 MVZ:Bird    |   426

```

Users:

 @bryansmclean
 @mkoo
 @campmlc
 @ccicero
 @mvzhuang
 @makaylaeasley
 @jtgiermakowski
 @wellerjes
 @keg34
 @AdrienneRaniszewski
 @acdoll
 @jldunnum
 @jrdemboski
 @msbparasites
 @ewommack
 @kderieg322079
 @catherpes
 @adhornsby
 @droberts49

_**Context**: Describe why this new value is necessary and existing values are not._

We seem to have two ways of doing the same thing.

_**Priority**: Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking._

Code table cleanup should always be highest priority.

@ArctosDB/arctos-code-table-administrators 

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality)._

- [x] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [x] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by dustymc** [comment]
I believe this was sufficiently approved by @ArctosDB/arctos-code-table-administrators and I would like to begin work on it TOMORROW.

@campmlc @ccicero @jldunnum please confirm and check some boxes.

**Comment by campmlc** [comment]
To be converted into examined for and not examined for attributes. Approved.

**Comment by dustymc** [comment]
Fresh data:

[temp_noexamfor_ecto.csv.zip](https://github.com/ArctosDB/arctos/files/11520376/temp_noexamfor_ecto.csv.zip)
[temp_examfor_ecto.csv.zip](https://github.com/ArctosDB/arctos/files/11520377/temp_examfor_ecto.csv.zip)
[temp_epa(1).csv.zip](https://github.com/ArctosDB/arctos/files/11520378/temp_epa.1.csv.zip)

done
